Generator.js

```diff
@@ -96,7 +101,6 @@ var generator = {
     // The `template` helper reads the specified template, making the
     // entire scope available to it (uses underscore/JST/ejs syntax).
     // Then the file is copied into the specified destination (on the left).
-    './api/hooks/role/index.js': { copy: 'hookShell.js' },
```

Comment:
This target will always be used and will exit in an error `error: Something else already exists at :: ~/my-sails-project/api/hooks/role/index.js`

I chose to add this target in the before method, like you already did for roleContext.